### ISSUE TITLE
Custom themes should be able to set round button corner radius, standard (usually square) button corner radius, and tile corner radius separately. More custom theme support ideas in here:

### ISSUE THREAD

**Issue by DrewNaylor** [DrewNaylor/Retiled#206]
- As stated in the title, options to set round button corner radiuses (usually they're circular, but they could be squared if the theme called for it), a different corner radius for the standard square/rectangular buttons, and a different corner radius for tiles
- An option for just solid-color background (besides white or black) with no image (without having to specify an image, setting the rectangle layer opacity to 100%, and turning off parallax scrolling)
- Some way to allow buttons to have 3D borders, a significantly-different looking "pressed" appearance, and shadows, if desired
- Something to allow stuff like buttons to have a glossy/glass-like texture if the theme calls for it
- Basically I want to allow anything from Aero Glass to Windows 9x-style (like the cool Windows 95 Mobile concept video: https://youtu.be/D0DDQumaaCg) themes, and more than anything that could possibly be imagined
- Maybe even iOS, Symbian, or something, I don't know
